src/datasets/utils/download_manager.py

```diff
@@ -228,6 +228,12 @@ def download_custom(self, url_or_urls, custom_download):
         Returns:
             downloaded_path(s): `str`, The downloaded paths matching the given input
                 url_or_urls.
+
+        Example:
+
+        ```py
+        >>> downloaded_files = dl_manager.download_custom('https://storage.googleapis.com/seldon-datasets/sentence_polarity_v1/rt-polaritydata.tar.gz')
```

Comment:
you need to pass a download function
```suggestion
        >>> downloaded_files = dl_manager.download_custom('s3://my-bucket/data.zip', custom_download_for_my_private_bucket)
```